src/gui/dialogs/end_credits.cpp

```diff
@@ -44,13 +44,15 @@ end_credits::end_credits(const std::string& campaign)
 	, text_widget_(nullptr)
 	, scroll_speed_(100)
 	, last_scroll_(std::numeric_limits<uint32_t>::max())
+	, first_idx_(0)
+	, last_idx_(first_idx_ + sliding_size_)
 {
 }
 
 void end_credits::pre_show(window& window)
 {
 	// Delay a little before beginning the scrolling
-	last_scroll_ = SDL_GetTicks() + 3000;
+	last_scroll_ = SDL_GetTicks();
```

Comment:
If the delay is removed now, should remove the comment explaining that delay as well.

Reply:
True.